ui/libs/search/src/lib/collections/services/adapter.ts

```diff
@@ -1,11 +1,12 @@
 import { IService } from './service.model';
-import { IResult } from '../../result.model';
+import {IResult} from "../../state";
 
 export const serviceToResult = (
-  service: Partial<IService>,
+  service: Partial<IService> & {id: string},
```

Comment:
The same as above. You're using `Partial<T> & { id: string }` in adapters, as also id in the models themselves.